src/etc/inc/plugins.inc.d/dhcpd.inc

```diff
@@ -480,16 +480,15 @@ function dhcpd_radvd_configure($verbose = false, $blacklist = array())
         $radvdconf .= "# Generated config for {$autotype} delegation from {$trackif} on {$if}\n";
         $radvdconf .= "interface {$realif} {\n";
         $radvdconf .= "\tAdvSendAdvert on;\n";
-        $radvdconf .= "\tMinRtrAdvInterval 3;\n";
-        $radvdconf .= "\tMaxRtrAdvInterval 10;\n";
         $radvdconf .= sprintf("\tAdvLinkMTU %s;\n", !empty($mtu) ? $mtu : 1280);
+        $radvdconf .= "\tAdvManagedFlag on;\n";
         $radvdconf .= "\tAdvOtherConfigFlag on;\n";
         $radvdconf .= "\tprefix {$networkv6} {\n";
         if ($autotype == 'slaac') {
             /* XXX also of interest in the future, see hardcoded prefix above */
             $radvdconf .= "\t\tBase6Interface $realtrackif;\n";
-            $radvdconf .= "\t\tDeprecatePrefix on;\n";
         }
+        $radvdconf .= "\t\tDeprecatePrefix on;\n";
```

Comment:
I know we set it for SLAAC when we included this but Is this wise to move to general use? Won't this cause intermittent connectivity issues?

Reply:
Well, according to previous work the choice we have to make is do we want to say we support CARP only for manual tracking use and go ahead with this. It might be worth adding VIPs to automatic case too and be done with the difference.

Reply:
It's always on in manual mode and on static interfaces, except for CARP(?) where it's always off.
It's always off in automatic mode, except for SLAAC WAN where it's always on.
We can have a discussion whether it should be on or off (or better: configurable), but I suggest it should at least be consistent.

> It might be worth adding VIPs to automatic case too and be done with the difference.

👍